### PR TITLE
Updates to the contribution guide

### PR DESCRIPTION
**Pull request by hickeyma**
Some small suggestions to the contribution guide to improve usability.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,20 +61,29 @@ Improvements to existing functionality are tracked as [GitHub issues using the U
 
 ### Set up your dev environment
 
-Your dev environment is set up by [tox](https://tox.wiki/en/latest/), an environment orchestrator which allows for setting up environments for and invoking builds, unit tests, formatting, linting, etc. Install tox with:
+The following tools are required:
+
+- [git](https://git-scm.com)
+- [python](https://www.python.org) (v3.9+)
```
Comment (evaline-ju) [inline CONTRIBUTING.md:67]: Q - we do also still support 3.8. is there a particular reason 3.9 is advised?

Comment (hickeyma) [reply CONTRIBUTING.md:67]: Good catch. Updated.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -22,7 +20,22 @@ Help on open source projects is always welcome and there is always something tha
...
+
+To contribute to this repo, you'll use the Fork and Pull model common in many open source repositories. For details on this process, watch [how to contribute](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github).
+
+When your contribution is ready, you can create a pull request. Pull requests are often referred to as "PR". In general, we follow the standard [github pull request](https://help.github.com/en/articles/about-pull-requests) process. Follow the template to provide details about your pull request to the maintainers.
+
+Before sending pull requests, make sure your changes pass formatting, linting and unit tests.
+
+#### Code Review
+
+Once you've [created a pull request](#how-to-contribute), maintainers will review your code and may make suggestions to fix before merging. It will be easier for your pull request to receive reviews if you consider the criteria the reviewers follow while working. Remember to:
```
[the remark this answers is not in the document]
Comment (hickeyma) [reply CONTRIBUTING.md]: Well spotted. Fixed.

**Comment by hickeyma** [comment]
Thanks for the review @evaline-ju. Ready for review again.

**Pull request merged**